src/core/meshcop/commissioner.cpp

```diff
@@ -258,10 +357,10 @@ otError Commissioner::SendCommissionerSet(void)
 
     dataset.mIsSteeringDataSet = true;
 
-    SuccessOrExit(error = SendMgmtCommissionerSetRequest(dataset, nullptr, 0));
+    IgnoreError(SendMgmtCommissionerSetRequest(dataset, nullptr, 0));
```

Comment:
Instead of ignoring the error, what about logging it?

Reply:
Updated it in the new pushed commit to log the error. 
We later probably need a mechanism on to handle the failure (retry, etc). Perhaps in a future PR.